Documentation/compatibility/wpf-KB-focus-now-moves-correctly-across-hosting-scenarios.md

```diff
@@ -0,0 +1,41 @@
+## Keyboard focus now moves correctly across multiple layers of WinForms/WPF hosting 
+
+### Scope
+Edge
+
+### Version Introduced
+4.7.2
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+Consider a WPF application hosting a WinForms control which in turn is hosting WPF controls. Users may not be able to tab out of the WinForms layer if the first or last control in that layer is the WPF ElementHost. This change fixes this issue, and users are now able to tab out of the WinForms layer, but automated applications that rely on focus looping inside the WinForms layer may be affected.
```

Comment:
is hosting --> hosts
ElementHost should be a link: \<xref:System.Windows.Forms.Integration.ElementHost>
"This change..." -- this is a run-on sentence. "but automated..." should be a separated sentence.
The "but automated..." is vague and unclear. What does affected mean in this context? Does automated looping in the WinForms layer no longer work?